back-end/db.js

```diff
@@ -28,6 +28,12 @@ const connect = async () => {
   console.log("Connected to database.");
 };
 
+const performQuery = async (query) => {
+  const result = await mssql.query(query);
```

Comment:
is het echt zo makkelijk?

Reply:
als alles er omheen werkt wel
aka
de rest van de file
de firewall op azure
db server op azure
db zelf op azure

lol